README.md

```diff
@@ -305,12 +305,20 @@ The following parameters are supported:
 - `order`: whether to order the transactions in ascending or descending order of occurrence. A value beginning with `d` or `D` is interpreted as descending; any other (or no) value is interpreted as ascending.
 - `from`: a transaction id. If the order is ascending, return transactions with higher ids than this; if the order is descending, return transactions with lower ids.
 - `limit`: the maximum number of transactions to return; defaults to 20; values above 1000 are treated as 1000.
-- `includeRewards`: whether to include rewards, and if so, which ones. This is
-  an optional parameter which defaults to including all rewards. The possible
-  values are
-  - `none`: include no rewards, including minting
-  - `allButFinalization`: include all but finalization rewards
-  - `all`: include all rewards. This is also the default if not supplied.
```

Comment:
This needs to be reverted as per the comment above. We cannot change the existing endpoint since we have to maintain backwards compatibility.